doc/active_docs/Account Management API.json

```diff
@@ -2647,10 +2644,7 @@
             "style": "form",
             "explode": false,
             "schema": {
-              "type": "array",
-              "items": {
-                "type": "object"
-              }
+              "$ref": "#/components/schemas/Permissions"
```

Comment:
`allowed_sections` has a predefined list of values, so an enum would be suitable here.

Reply:
not clear with this point

Reply:
done